aurora/integration-tests/src/lib.rs

```diff
@@ -32,10 +32,12 @@ mod tests {
     const NEAR_DEPOSIT: u128 = 2 * near_sdk::ONE_NEAR;
     const WNEAR_FOR_TOKENS_TRANSFERS: u128 = 100 * near_sdk::ONE_YOCTO;
 
+    const TRANSFER_EXPIRATION_PERIOD_S: u64 = 30;
```

Comment:
```suggestion
    const TRANSFER_EXPIRATION_PERIOD_SEC: u64 = 30;
```

Reply:
IMO, the `_S` suffix is sufficient enough. E.g., if we use milliseconds, we would use `_MS`. But it's a matter of taste.

Reply:
fixed: https://github.com/aurora-is-near/fast-bridge-protocol/pull/101/commits/fc520f211b971e073aae04aa32c26491e606f6d5